Makefile

```diff
@@ -133,6 +133,13 @@ test: manifests generate fmt vet envtest ## Run tests.
 test-e2e:
 	go test ./test/e2e/ -v -ginkgo.v
 
+.PHONY: markdownlint
+markdownlint:
+	@$(CONTAINER_TOOL) run --platform linux/amd64 -v .\:/md\:z  docker.io/peterdavehello/markdownlint  markdownlint --dot --ignore ./vendor --dot --config .markdownlint.yaml ./
```

Comment:
lets have a var for container image to avoid duplication

Reply:
good idea. done